### PR TITLE
feat(icons): new omnichannel brand icons

### PR DESCRIPTION
**Pull request by tedgoas-dialpad**
## Description
Adding brand icons for future omnichannel integrations:
Telegram, WeChat, Google Business Messaging, LINE, Viber, TikTok, Snapchat

DP-49152

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)
giphy-2

### CODE REVIEW

File: docs/_data/svg-brand.json [hunk 1]
```diff
@@ -287,12 +317,24 @@
...
   {
     "name": "Visa",
     "file": "visa",
     "vue": "IconVisa",
     "desc": "Visa logo"
   },
+  {
+    "name": "WeChat",
+    "file": "wechat",
+    "vue": "IconWeChat",
```
Comment (braddialpad) [inline docs/_data/svg-brand.json:335]: The WeChat icon isn't showing up in the preview. The PascalCase name here is derived from the svg filename on build capitalizing the first character after the `-`. To fix this either:

1. Change this line to `IconWechat`
2. Change the svg file name to `we-chat.svg`

**Review by braddialpad — CHANGES_REQUESTED** [review]

File: docs/_data/svg-brand.json (hunk 1 above)
Comment (tedgoas-dialpad) [reply docs/_data/svg-brand.json:335]: Ahh, I didn't know that's how it worked! Just pushed the change.

**Comment by braddialpad** [comment]
Looks like you updated the json correctly, but missed fixing the filename itself (from wechat.svg to we-chat.svg)

**Review by braddialpad — APPROVED** [review]
Looks great!

**Pull request merged**